__tests__/Product.native.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { create } from "react-test-renderer";
+import { Product } from "../Components/Product.native";
+import products from "../Fixtures/products.json";
+
+const product = products[0];
+
+test('renders correctly with no data', () => {
+  const tree = create(<Product />).toJSON();
+
+  expect(tree).toMatchSnapshot();
+  expect.objectContaining({Text:"Opps! No Data!"})
+});
+
+test('renders correctly with undefined data', () => {
+  const tree = create(<Product product={undefined}/>).toJSON();
+
+  expect(tree).toMatchSnapshot();
+  expect.objectContaining({Text:"Opps! No Data!"})
+});
+
+test('renders correctly with data', () => {
+  const tree = create(<Product product={product} />).toJSON();
+
+  expect(tree).toMatchSnapshot();
+  expect.objectContaining({Text:"Chanel"})
+});
+
+// TODO: Should be testing this method directly but keep getting mock function errors.
```

Comment:
I don't like not being able to test the method directly, but I tried a few different ways to test it after importing it and I kept getting errors, so I figured mocking the function was better than nothing, but it feels silly since it's not testing the actual code, nor is it the same...

Reply:
@maribies I'm not sure this test is testing anything at all. If you change the real function does this fail? If you make a breaking change to your prod code and the test still passes, it's likely a bad test.

Reply:
i'm going to remove this then